From f7e6d67242ee2a8ee8761a62c94079c67432a6bb Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sat, 28 May 2022 11:47:32 -0400
Subject: [PATCH 1855/1936] Disable external rtosc CI build

Another rtosc release is needed to restore the previous arch linux
devendored workflow.
---
 .github/workflows/ccpp.yml | 48 +++++++++++++++++++-------------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/.github/workflows/ccpp.yml b/.github/workflows/ccpp.yml
index 60cce343..57da16de 100644
--- a/.github/workflows/ccpp.yml
+++ b/.github/workflows/ccpp.yml
@@ -25,27 +25,27 @@ jobs:
       run: cd build && make
     - name: make test
       run: cd build && make test
-  build_arch_linux_system_rtosc:
-    runs-on: ubuntu-latest
-    container:
-      image: archlinux:latest
-    steps:
-      - name: Install dependencies
-        run: pacman --noconfirm -Syu alsa-lib base-devel cmake cxxtest dssi fftw fltk git jack ladspa liblo libxpm mxml portaudio rtosc zlib
-      - uses: actions/checkout@v2
-        with:
-          submodules: recursive
-      # test does not seem to work in a container and reports:
-      # Process not started
-      #  /__w/zynaddsubfx/zynaddsubfx/build/src/Tests/check-ports.rb
-      # [no such file or directory]
-      - name: Disable PortChecker test
-        run: sed -e '/add_test(PortChecker/d' -i src/Tests/CMakeLists.txt
-      - name: Build zynaddsubfx
-        run: |
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DZYN_SYSTEM_RTOSC=ON -B build -S .
-          make VERBOSE=1 -C build
-      - name: Test zynaddsubfx
-        run: make VERBOSE=1 -k test ARGS='-VVd' -C build
-      - name: Install zynaddsubfx
-        run: make VERBOSE=1 install -C build
+#  build_arch_linux_system_rtosc:
+#    runs-on: ubuntu-latest
+#    container:
+#      image: archlinux:latest
+#    steps:
+#      - name: Install dependencies
+#        run: pacman --noconfirm -Syu alsa-lib base-devel cmake cxxtest dssi fftw fltk git jack ladspa liblo libxpm mxml portaudio rtosc zlib
+#      - uses: actions/checkout@v2
+#        with:
+#          submodules: recursive
+#      # test does not seem to work in a container and reports:
+#      # Process not started
+#      #  /__w/zynaddsubfx/zynaddsubfx/build/src/Tests/check-ports.rb
+#      # [no such file or directory]
+#      - name: Disable PortChecker test
+#        run: sed -e '/add_test(PortChecker/d' -i src/Tests/CMakeLists.txt
+#      - name: Build zynaddsubfx
+#        run: |
+#          cmake -DCMAKE_INSTALL_PREFIX=/usr -DZYN_SYSTEM_RTOSC=ON -B build -S .
+#          make VERBOSE=1 -C build
+#      - name: Test zynaddsubfx
+#        run: make VERBOSE=1 -k test ARGS='-VVd' -C build
+#      - name: Install zynaddsubfx
+#        run: make VERBOSE=1 install -C build
-- 
2.47.0

